flows/headers/tcp/test_tcp_header.py

```diff
@@ -0,0 +1,132 @@
+import logging as log
+import pytest
+from helpers.otg import otg
+
+
+@pytest.mark.all
+@pytest.mark.feature
+@pytest.mark.b2b
+@pytest.mark.free_feature
+def test_tcp_header():
+    test_const = {
+        "pktRate": 50,
+        "pktCount": 100,
+        "pktSize": 128,
+        "txMac": "00:00:01:01:01:01",
+        "rxMac": "00:00:01:01:01:02",
+        "txIp": "1.1.1.1",
+        "rxIp": "1.1.1.2",
+        "txTcpPort": 5000,
+        "rxTcpPort": 6000,
+    }
+
+    api = otg.OtgApi()
+    c = tcp_header_config(api, test_const)
+
+    api.set_config(c)
+
+    api.start_capture()
+    api.start_transmit()
+
+    api.wait_for(
+        fn=lambda: metrics_ok(api, test_const), fn_name="wait_for_flow_metrics"
+    )
+
+    api.stop_capture()
+
+    capture_ok(api, c, test_const)
+
```

Comment:
Add a check to validate pktSize on received frames

P.S: Add the check for all the other py tests and go tests